examples/hg-ReferralComposition-01.xml

```diff
@@ -25,76 +25,86 @@
-        </text>
-        <entry>
-            <reference value="Condition/hg-ReferralServiceRequest-01-Condition-01"/>
-            <type value="Condition"/>
-            <display value="L15 Knie symptomen/klachten"/>
-        </entry>
+        <section>
+            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-TextValue">
+                <valueString value="Ik verwijs mevrouw voor knieklachten links. Sinds 3 maanden ontstaan na periode van overbelasting. Beeld mijns inziens passend bij gonartrose."/>
+            </extension>
+            <title value="Reden bericht"/>
+            <code>
+                <coding>
+                    <system value="http://snomed.info/sct"/>
+                    <code value="440378000"/>
+                    <display value="Referral for (observable entity)"/>
+                </coding>
+            </code>
+            <text>
+                <status value="extensions"/>
+                <div xmlns="http://www.w3.org/1999/xhtml">Reden bericht: L15 Knie symptomen/klachten, Ik verwijs mevrouw voor knieklachten links. Sinds 3 maanden ontstaan na periode van overbelasting. Beeld mijns inziens passend bij gonartrose.</div>
+            </text>
+            <entry>
+                <reference value="Condition/hg-ReferralServiceRequest-01-Condition-01"/>
+                <type value="Condition"/>
+                <display value="L15 Knie symptomen/klachten"/>
+            </entry>
+        </section>
+        <section>
+            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-TextValue">
+                <valueString value="paracetamol indien nodig"/>
+            </extension>
+            <title value="Ingestelde behandeling"/>
+            <code>
+                <coding>
+                    <system value="http://snomed.info/sct"/>
+                    <code value="108631000146107"/>
+                    <display value="behandeling lopende (situatie)"/>
+                </coding>
+            </code>
+            <text>
+                <status value="extensions"/>
+                <div xmlns="http://www.w3.org/1999/xhtml">Ingestelde behandeling: paracetamol indien nodig</div>
+            </text>
+        </section>
+        <section>
+            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-TextValue">
+                <valueString value="graag uw fysiotherapeutische diagnose en behandeling, indien na 3 maanden geen verbetering retour voor eventuele doorverwijzing naar orthopeed"/>
+            </extension>
+            <title value="Procedurevoorstel"/>
+            <code>
+                <coding>
+                    <system value="http://snomed.info/sct"/>
+                    <code value="240471000146100"/>
+                    <display value="vertaling toelichting op aard van verwijzing (waarneembare entiteit)"/>
```

Comment:
Use 'toelichting op aard van verwijzing (waarneembare entiteit)' or 'toelichting op aard van verwijzing' (I'd prefer the latter).